packages/ingest/granule.js

```diff
@@ -321,19 +321,23 @@ class Granule {
   }
 
   /**
-   * Filter out md5 checksum files and put them in `this.checksumFiles` object.
+   * Filter out checksum files and put them in `this.checksumFiles` object.
    * To be used with `Array.prototype.filter`.
    *
    * @param {Object} file - file object from granule.files
-   * @returns {boolean} depending on if file was an md5 checksum or not
+   * @returns {boolean} - whether file was a supported checksum or not
    */
   filterChecksumFiles(file) {
-    if (file.name.indexOf('.md5') > 0) {
-      this.checksumFiles[file.name.replace('.md5', '')] = file;
-      return false;
-    }
+    let unsupported = true;
+    this.supportedChecksumFileTypes.forEach((type) => {
```

Comment:
Good change. Is there any reason it wasn't doing this already?

Reply:
It seemed to assume `md5` checksums in a lot of cases. We're still limited to the 4 checksums I added in `supportedChecksumFileTypes` and default to md5 if none match but it's a better situation than assuming`md5`. I didn't know how else to filter out checksum files, since the only difference seems to be that they have an extra extension with the name of the algorithm.

Reply:
If there are other common checksum algorithms in the earth science space I'm open to adding them.